plugins/ipv4net/host.go

```diff
@@ -170,9 +174,20 @@ func (n *IPv4Net) interconnectAfpacket() (key string, config *interfaces.Interfa
 		Mtu:         n.ContivConf.GetInterfaceConfig().MTUSize,
 		Enabled:     true,
 		Vrf:         n.ContivConf.GetRoutingConfig().MainVRFID,
-		IpAddresses: []string{n.IPAM.HostInterconnectIPInVPP().String() + "/" + strconv.Itoa(size)},
 		PhysAddress: hwAddrForNodeInterface(n.NodeSync.GetNodeID(), hostInterconnectHwAddrPrefix),
 	}
+	if n.ContivConf.InSTNMode() {
+		afpacket.Unnumbered = &interfaces.Interface_Unnumbered{
+			InterfaceWithIp: n.ContivConf.GetMainInterfaceName(),
+		}
+	} else {
+		afpacket.IpAddresses = []string{n.IPAM.HostInterconnectIPInVPP().String() + "/" + strconv.Itoa(size)}
+	}
+	if n.ContivConf.GetInterfaceRxMode() != interfaces.Interface_RxModeSettings_POLLING {
```

Comment:
yeah, but I would not explicitly configure the `Interface_RxModeSettings_DEFAULT` - vpp-agent now returns an error by attempt to configure it on the DPDK interface (easy to fix), but I doubt that it would be dumped as `Interface_RxModeSettings_DEFAULT` from VPP - it would always dump the current setting - polling / interrupt / adaptive IMHO. We would also save one useless binary API call per interface.